roles/k3s_prep/tasks/arm.yml

```diff
@@ -0,0 +1,8 @@
+---
+- name: Copy k3s images into place
+  ansible.builtin.get_url:
+    url: "https://github.com/k3s-io/k3s/releases/download/v1.28.2%2Bk3s1/k3s-airgap-images-arm.tar.gz"
```

Comment:
Please variablize the `v1.28.2%2Bk3s1`.

You can use URLEncode to convert a human-friendly string: https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_filters.html#urlencode-variables